client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -525,7 +525,7 @@ class SiteSettingsFormDiscussion extends Component {
 		} = this.props;
 		return (
 			<FormFieldset>
-				<FormLegend>{ translate( 'Comment Blacklist' ) }</FormLegend>
+				<FormLegend>{ translate( 'Comment blacklist' ) }</FormLegend>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.